client/spec/authoring_spec.js

```diff
@@ -102,4 +102,29 @@ describe('authoring', function() {
         });
 
     });
+
+    describe('subnavigation menu', function () {
+        it('allows to create a new empty package', function () {
+            // 1. Navigate to the authoring main page.
+            //
+            // 2. Open the "plus icon" menu in subnav bar and click the
+            //    "Empty Package" link in the menu.
+            //
+            // 3. Get the newly created item's info box and check if it
+            //    contains an icon indicating that the item's type is indeed
+            //    "composite".
+            //
+            authoring.openAuthoringHome();
+
+            $('.navbtn .sd-create-btn').click();
```

Comment:
Better to user PageObjects style to write the tests, see:
http://www.protractortest.org/#/page-objects